frontend/website/src/Render.re

```diff
@@ -146,6 +146,18 @@ Router.(
             <Blog posts />
           </Page>,
         ),
+        File(
+          "privacy",
+          <Page name="privacy">
```

Comment:
You should make this a legacy page if you're using tachyons stuff to style it. I imagine we'll get Page off of tachyons at some point.

Reply:
right good point